snd-server/app.py

```diff
@@ -34,6 +40,12 @@ def create_app(ctx: Context) -> FastAPI:
     app.include_router(create_health_router(ctx), prefix="/api")
     app.include_router(create_tracks_router(ctx), prefix="/api")
 
+    provider = TracerProvider()
+    processor = BatchSpanProcessor(OTLPSpanExporter())
+    provider.add_span_processor(processor)
+    trace.set_tracer_provider(provider)
+    tracer = trace.get_tracer("soundations")
```

Comment:
you could use less generic var names (`tracer_provider` instead of `provider`) or move this to a separate function that would initialize everything and only return what is needed outside (`tracer` maybe?)

Also might make sense to put some of it inside `ctx` to be able to add some data during request processing, but not sure about this one.


This is optional and we can do it later when the use case is clear :)

Reply:
I would rather do it later when we add some more tracers or spans :)